akd_core/src/build.rs

```diff
@@ -14,23 +14,24 @@
 const PROTOBUF_BASE_DIRECTORY: &str = "src/proto/specs";
 /// The list of protobuf files to generate inside PROBUF_BASE_DIRECTORY
 const PROTOBUF_FILES: [&str; 1] = ["types"];
+/// The output directory in the cargo build folder to emit the generated sources to
+const OUTPUT_RELATIVE_DIR: &str = "protos";
```

Comment:
Super tiny nit: This still feels quite specific to protos despite being in a seemingly generic location. I think a better name would be `PROTOS_OUTPUT_DIR` (can include relative if you'd like).